app/generated_resources.grd

```diff
@@ -4744,6 +4753,12 @@ Keep your key file in a safe place. You will need it to create new versions of y
       <message name="IDS_UPDATE_PASSWORD_DIFFERENT_DOMAINS_TITLE" desc="The title of the update password bubble when the submitted form origin isn't equal to saved credentials origin.">
         Update password for <ph name="ORIGIN">$1<ex>example.com</ex></ph>?
       </message>
+      <message name="IDS_SAVE_PASSWORD_TO_GOOGLE" desc="The title of the save password bubble when the user syncs passswors to thir Brave account.">
```

Comment:
I don't understand what are these new strings, maybe we need to disable some Chromium UI stuff.   Building to look on it.

Reply:
these strings are not used because the `MessagesForAndroidInfrastructure` and `MessagesForAndroidPasswords` are disabled, https://source.chromium.org/chromium/chromium/src/+/master:chrome/browser/password_manager/chrome_password_manager_client.cc;l=342;drc=1f60f34982eb1058a74b77fcc71ffdba05990771;bpv=1;bpt=1